examples/message-metadata/app.js

```diff
@@ -0,0 +1,55 @@
+const { App, LogLevel } = require('@slack/bolt');
+
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  appToken: process.env.SLACK_APP_TOKEN,
+  socketMode: true,
+  logLevel: LogLevel.DEBUG
+});
+
+(async () => {
+  await app.start();
+  console.log('⚡️ Bolt app started');
+})();
+
+
+// Listen to slash command
+// Post a message with Message Metadata
+app.command('/post', async ({ ack, command, say }) => {
+  await ack();
+  say({
```

Comment:
```suggestion
  await say({
```

Reply:
My mistake, thanks for catching. This is fixed.